### ISSUE TITLE
Consider "/" as the project root directory, always

### ISSUE THREAD

**Issue by salbeira** [decker-edu/decker#109]
Right now prefixing a path with a leading "/" does not have consistent behavior. One thing that especially bothers me is that links do not get any attention from our "turn paths to relative paths" mechanism that is applied to media elements.

Right now we consider paths without a relative prefix (no `./`or `../`) to be leading from the project root:

`images/image.png` gets treatment, `/images/image.png` gets this too when linked in a media-element, but in a meta variable the latter sometimes fails:

```
logos:
    - '/images/logo.png'
``` 

gets no conversion while

```
logos:
    - 'images/logo.png'
```

does.

Curious enough re-adding the `logos` variable to the `runtime-path-variable` inside the `decker.yaml` makes the first version work even though the list of runtime-path-variables does not change. I already presented this bug to @monofon .

I doubt there is ever a situation where one would link to a file in their linux-system-root or wishes to reference something from the web-root of their host. At least these are the situations where a "/" prefix would have a different meaning than "reference a file relative to the project root".

I propose that every URI, be it in a meta variable, a media element or a link that begins with a "/" gets our "relative path treatment" and references a file starting from the project root.

Examples:

### index.md

```
[Slides](/decks/introduction.html) -> <a href="./decks/introduction.html">Slides</a>
```

### /decks/introduction.html

```
[Next Chapter](/decks/advanced.html) -> <a href="./advanced.html">Next Chapter</a>
```

### /decks/advanced.html

```
[Back to the Index](/index.html) -> <a href="../index.html">Back to the Index</a>
```

**Comment by monofon** [comment]
Ja, makes absolute sense. This is how it should have been from the beginning.

Postponed until after the next milestone.
